File: diff.py
def get_ages(sum_, difference):
    half = sum_ / 2
    diff = difference / 2
    if half < 0 or difference < 0:
        return None
    else:
        one = float( half + diff)
        two = float(half - diff)
    if one < 0 or two < 0:
        return None
    else:
        return (one, two)

File: diff.js
function getAges(sum,difference){
    if (sum < 0 || difference <0 ){
      return null
    }
    let sumHalf = sum / 2
    let difHalf = difference / 2
    let one = sumHalf + difHalf
    let two = sumHalf - difHalf
    if (one < 0 || two < 0){return null}
    return [one, two]
  };